Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocColumnRendererBase.cs

```diff
@@ -104,12 +104,35 @@ protected virtual bool HasContentAttribute
       get { return false; }
     }
 
+    /// <summary>
+    /// Provides secure forwarding of additional attributes to  <see cref="BocColumnRenderingContext"/>
+    /// </summary>
+    public struct BocRenderArguments
```

Comment:
I think `BocRenderArguments` is a bit misleading. Mike proposed `BocDataCellAttributeRenderingContext` and `BocTitleCellAttributeRenderingContext` but since we only have one struct would use `BocCellAttributeRenderingContext`.

We also put this struct into its own file. From mikes comment:

> [...] not nested, we're in a specific namespace anyway and the other types aren't nested either.